include/deal.II/lac/sparsity_pattern_base.h

```diff
@@ -106,6 +108,22 @@ class SparsityPatternBase : public Subscriptor
   add_entries(const ArrayView<const size_type> &rows,
               const ArrayView<const size_type> &columns);
 
+  /**
+   * General function for adding new entries.
+   *
+   * This function requires a mutable array view since it sorts the input by row
+   * and column indices. It is most useful when assembling a sparsity pattern
```

Comment:
Would it add noticeable cost to just copy the entries into a `std::vector` and sort that?